BrainPortal/app/models/tool.rb

```diff
@@ -58,30 +58,25 @@ class Tool < ActiveRecord::Base
   has_many                :bourreaux, :through => :tool_configs, :uniq => true
 
   attr_accessible         :name, :user_id, :group_id, :category, :license_agreements,
-                          :cbrain_task_class, :select_menu_text, :description, :url,
+                          :cbrain_task_class_name, :select_menu_text, :description, :url,
                           :application_type, :application_tags, :application_package_name
 
 
   # CBRAIN extension
   force_text_attribute_encoding 'UTF-8', :description
 
-  # Find a random bourreau on which this tool is available and to which +user+ has access.
-  def select_random_bourreau_for(user)
-    available_group_ids = user.group_ids
-    bourreau_list = Bourreau.where( :group_id => available_group_ids, :online => true ).select(&:is_alive?)
-    bourreau_list = bourreau_list & self.bourreaux
-    if bourreau_list.empty?
-      cb_error("Unable to find an execution server. Please notify your administrator of the problem.")
-    end
-    bourreau_list.slice(rand(bourreau_list.size)).id
-  end
-
```

Comment:
Yes, it was a method used by the old interface, and not used at all anymore anywhere.